### ISSUE TITLE
HMM Voice building

### ISSUE THREAD

**Issue by yash90** [marytts/marytts#422]
Hi,
While building my HMM based voice I am getting an error in the "HMMVoiceMakeVoice" step as follows.

```
Error in /home/yashj/Mary/marytts/lib/external/bin/HERest    -A    -C /home/yashj/Mary/HMM_Voice_si/hts/configs/trn.cnf -D -T 1 -S /home/yashj/Mary/HMM_Voice_si/hts/data/scp/train.scp -I /home/yashj/Mary/HMM_Voice_si/hts/data/labels/mono.mlf -m 1 -u tmvwdmv -w 5000 -t 1500 100 5000  -H /home/yashj/Mary/HMM_Voice_si/hts/models/qst001/ver1/cmp/monophone.mmf -N /home/yashj/Mary/HMM_Voice_si/hts/models/qst001/ver1/dur/monophone.mmf -M /home/yashj/Mary/HMM_Voice_si/hts/models/qst001/ver1/cmp -R /home/yashj/Mary/HMM_Voice_si/hts/models/qst001/ver1/dur /home/yashj/Mary/HMM_Voice_si/hts/data/lists/mono.list /home/yashj/Mary/HMM_Voice_si/hts/data/lists/mono.list
```
I installed required programmes using check_install_external_programs.sh. Can anyone help me ?

**Comment by seblemaguer** [comment]
Can you indicate a little bit more ? Apparently it's not a software installation problem but something else and I can't help you with little error message.

**Comment by yash90** [comment]
I followed steps as given in marytts/wiki. When I run HMMVoiceMakeVoice  step, it runs for 2-3 hours and then stops giving above error in the terminal. I used perl scripts/Training.pl scripts/Config.pm as instructed. 
These are the last lines of the log file.
HMM:
HERest  ML Updating: Transitions Means Variances MixWeights 
 System is PLAIN
52 Logical/52 Physical Models Loaded, VecSize=123
1 MMF input files
Duration model:
 HERest  ML Updating: Means Variances 
 System is PLAIN
 52 Logical/52 Physical Models Loaded, #States=5
 1 MMF input files

Pruning-On[1500.0 100.0 5000.0]

 Processing Data: s20001.cmp; Label s20001.lab
 Utterance prob per frame = 8.771466e+01
 Processing Data: s20002.cmp; Label s20002.lab
 Utterance prob per frame = 8.540571e+01
 Processing Data: s20003.cmp; Label s20003.lab
 Utterance prob per frame = 8.054584e+01
 Processing Data: s20004.cmp; Label s20004.lab
 Utterance prob per frame = 9.439617e+01
 Processing Data: s20005.cmp; Label s20005.lab
 Utterance prob per frame = 8.887272e+01
 Processing Data: s20006.cmp; Label s20006.lab
 Utterance prob per frame = 9.319072e+01
 Processing Data: s20007.cmp; Label s20007.lab
 Utterance prob per frame = 9.305796e+01
 Processing Data: s20008.cmp; Label s20008.lab

There are 1001 sentences.
